packages/hub-score/src/app.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
+    return 'Hello World!';
```

Comment:
hello ?

Reply:
the route was deleted

Reply:
i a default service method that generate when you create a nestjs project